tools/test

```diff
@@ -1,4 +1,21 @@
 #!/usr/bin/env bash
+
+# Careful! This doesn't do everything you'd think it does. In fact, we
```

Comment:
I would replace `This` here with `'set -e'`, since otherwise it's unclear what this comment is referring to until you read the entire thing.